src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/GetAzureStorSimpleResource.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.WindowsAzure.Commands.StorSimple.Properties;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple.Cmdlets
+{
+    /// <summary>
+    /// this commandlet returns all resources available in your subscription
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureStorSimpleResource"), OutputType(typeof(IEnumerable<ResourceCredentials>), typeof(ResourceCredentials))]
+    public class GetAzureStorSimpleResource : StorSimpleCmdletBase
+    {
+        [Alias("Name")]
```

Comment:
FYI, from cmdlets desin perspective, it's confusing to use same alias for different parameters. Now, Name sometimes match Device and other times Resource